test/integration/integration_test.exs

```diff
@@ -49,13 +49,21 @@ defmodule KafkaEx.Integration.Test do
     assert consumer_group_update_interval == 30000
   end
 
-  test "create_worker provides a default consumer_group of 'kafka_ex'" do
```

Comment:
Providing a default `consumer_group` of `kafka_ex` allows us to magically manage for users of the library, which makes kafka_ex work out of the box with little intervention. I prefer to make disabling `consumer_group` a second class thing so you have to explicitly not want it